src/Bridges/indicatorbridge.jl

```diff
@@ -0,0 +1,48 @@
+
+"""
+    IndicatorActiveOnFalseBridge{T}
+
+The `IndicatorActiveOnFalseBridge` replaces an indicator constraint activated on 0
+with the constraint activated on 1, creating an additional variable.
+"""
+struct IndicatorActiveOnFalseBridge{F <: MOI.AbstractScalarFunction, S <: MOI.AbstractScalarSet, T <: Real} <: AbstractBridge
+    variable_index::MOI.VariableIndex
+    zero_one_cons::MOI.ConstraintIndex{MOI.SingleVariable, MOI.ZeroOne}
+    disjunction_cons::MOI.ConstraintIndex{MOI.ScalarAffineFunction{T}, MOI.EqualTo{T}}
+    indicator_cons_index::MOI.ConstraintIndex{F, MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE, S}}
+end
+
+function IndicatorActiveOnFalseBridge(model::MOI.ModelLike, f::MOI.ScalarAffineFunction{T}, s::IS) where {T <: Real, F, IS <: MOI.IndicatorSet{MOI.ACTIVATE_ON_ZERO, S} where S <: MOI.AbstractScalarSet}
+    z1 = f.terms[1].variable_index
```

Comment:
Do `fs = MOIU.eachscalar(f)` and then `z1 = conver(MOI.SingleVariable, fs[1]).variable`